src/handlers/addTag.ts

```diff
@@ -0,0 +1,40 @@
+import { NextFunction, Request, Response } from 'express';
+import createHttpError from 'http-errors';
+import HttpStatusCodes from 'http-status-codes';
+import { db } from '../db';
+import { isNilOrEmpty } from '../util/isNilOrEmpty';
+
+export const addTagHandler = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const { id } = req.params;
+    const { tags } = req.body;
```

Comment:
you should definitely parse/validate this payload here to ensure this is an array of string only